packages/pods-core/src/v2/pods/export/JSONExportPodV2.ts

```diff
@@ -0,0 +1,73 @@
+import {
+  DendronError,
+  NoteProps,
+  ResponseUtil,
+  RespV2,
+} from "@dendronhq/common-all";
+import { JSONSchemaType } from "ajv";
+import _ from "lodash";
+import path from "path";
+import fs from "fs-extra";
+import { ConfigFileUtils, ExportPodV2 } from "../../..";
+import { JSONV2PodConfig, RunnableJSONV2PodConfig } from "../..";
+
+/**
+ * Markdown Export Pod (V2 - for compatibility with Pod V2 workflow).
+ */
+
+export type JSONExportReturnType = RespV2<{
+  exportedNotes?: string | NoteProps[];
```

Comment:
we can remove `string` now right?

Reply:
The string here is the return type when the destination is clipboard(equivalent to JSON publish pod)